### DISCUSSION TITLE
i965 Enabling

### DISCUSSION THREAD

**Discussion by StarMonkey1** [blakeblackshear/frigate#6653] [Ask A Question]
I've added the following to my docker-compose.
```
    environment:
      FRIGATE_RTSP_PASSWORD: "password"
      LIBVA_DRIVER_NAME: i965
```
What do I need to do next to ensure my Ubuntu 22.04 OS is utilising the driver?

**Comment by NickM-27** [comment]
You can run vainfo from the webUI to confirm, but that is all it takes

**Reply by StarMonkey1** [reply]
Ok, so this looks like I'm successfully running the i965 driver.

```{"return_code":0,"stderr":"","stdout":"vainfo:VA-APIversion:1.17(libva2.10.0)nvainfo:Driverversion:Inteli965driverforIntel(R)GeminiLake-2.4.1nvainfo:SupportedprofileandentrypointsnVAProfileMPEG2Simple:tVAEntrypointVLDnVAProfileMPEG2Main:tVAEntrypointVLDnVAProfileH264ConstrainedBaseline:tVAEntrypointVLDnVAProfileH264ConstrainedBaseline:tVAEntrypointEncSlicenVAProfileH264ConstrainedBaseline:tVAEntrypointEncSliceLPnVAProfileH264Main:tVAEntrypointVLDnVAProfileH264Main:tVAEntrypointEncSlicenVAProfileH264Main:tVAEntrypointEncSliceLPnVAProfileH264High:tVAEntrypointVLDnVAProfileH264High:tVAEntrypointEncSlicenVAProfileH264High:tVAEntrypointEncSliceLPnVAProfileH264MultiviewHigh:tVAEntrypointVLDnVAProfileH264MultiviewHigh:tVAEntrypointEncSlicenVAProfileH264StereoHigh:tVAEntrypointVLDnVAProfileH264StereoHigh:tVAEntrypointEncSlicenVAProfileVC1Simple:tVAEntrypointVLDnVAProfileVC1Main:tVAEntrypointVLDnVAProfileVC1Advanced:tVAEntrypointVLDnVAProfileNone:tVAEntrypointVideoProcnVAProfileJPEGBaseline:tVAEntrypointVLDnVAProfileJPEGBaseline:tVAEntrypointEncPicturenVAProfileVP8Version0_3:tVAEntrypointVLDnVAProfileVP8Version0_3:tVAEntrypointEncSlicenVAProfileHEVCMain:tVAEntrypointVLDnVAProfileHEVCMain:tVAEntrypointEncSlicenVAProfileHEVCMain10:tVAEntrypointVLDnVAProfileHEVCMain10:tVAEntrypointEncSlicenVAProfileVP9Profile0:tVAEntrypointVLDnVAProfileVP9Profile0:tVAEntrypointEncSlicenVAProfileVP9Profile2:tVAEntrypointVLD"}```
